cmc/values.yaml

```diff
@@ -19,6 +19,8 @@ global:
   ccdCaseDataApiUrl: ""
   togglesUrl: ""
   postgresqlHost: ""
+  jddoEnabled: true
+  laDirectionOrderTemplate: "CV-CMC-GOR-ENG-0004.docx"
```

Comment:
Please move this back to application.yaml where other relevant fields belong. As this is going to be constant for all envs.
```
doc_assembly:
  url: ${DOC_ASSEMBLY_URL:false}
  templateId: "CV-CMC-GOR-ENG-0004.docx"
  judgeTemplateId: "CV-CMC-GOR-ENG-00219.docx"
<put it here pelase>
```

Reply:
It is not consistent. There are breaking changes so the templates are now versioned and need to be changed on a per environment basis